doc/build/sysbuild/index.rst

```diff
@@ -0,0 +1,370 @@
+.. _sysbuild:
+
+Sysbuild (System build)
+#######################
+
+Sysbuild is a higher level CMake and Kconfig based build system which support
+configuring and building a Zephyr application together with additional projects,
+both Zephyr based but also non-Zephyr based projects.
+
+A common use-case is building a Zephyr application together with MCUboot.
+
+Architectural Overview
+**********************
+
+Architectural overview of the sysbuild project.
+
+Sysbuild itself natively supports building of a Zephyr application including
+MCUboot.
+
+Additional applications may be specified by the application or the
+board. Those applications can be Zephyr based, but it is also possible to add
+non-Zephyr based projects to sysbuild build infrastructure.
+
+.. figure:: sysbuild.svg
+    :align: center
+    :alt: Sysbuild architectural overview
+    :figclass: align-center
+    :width: 80%
+
+Building an application with sysbuild
+*************************************
+
+Sysbuild is a CMake project on higher level than the Zephyr CMake build system.
+This means that sysbuild is built using the same tools as a regular Zephyr CMake
+application.
+
+Using west:
+
+.. zephyr-app-commands::
+   :tool: west
+   :app: samples/hello_world
+   :board: reel_board
+   :goals: build
+   :west-args: --sysbuild
+   :compact:
+
+.. tip::
+
+   Sysbuild can also build single image builds, and it is possible to configure
+   ``west build`` to use ``--sysbuild`` per default for both single and
+   multi-image builds. This allows you to always build through sysbuild for all
+   samples without considering if they are single or multi-image based samples.
+
+   Execute ``west config build.sysbuild True`` to use sysbuild per default.
+   The ``west build --no-sysbuild ...`` can later be used as build flag when
+   sysBuild shall not be used.
```

Comment:
*sysbuild should

Reply:
Done, addressed by https://github.com/zephyrproject-rtos/zephyr/pull/43846#pullrequestreview-1060453559